client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/buffer/SortBasedDataBuffer.java

```diff
@@ -262,15 +263,15 @@ private void updateWriteSegmentIndexAndOffset(int numBytes) {
   }
 
   @Override
-  public BufferWithChannel copyIntoSegment(
-      MemorySegment target, BufferRecycler recycler, int offset) {
+  public BufferWithSubpartition getNextBuffer(
+      MemorySegment transitBuffer, BufferRecycler recycler, int offset) {
     checkState(hasRemaining(), "No data remaining.");
     checkState(isFinished, "Should finish the sort buffer first before coping any data.");
```

Comment:
Many other places in the comments show `sort buffer` or `Sort buffer`, we can fix them together.